tools/pyi/gen_pyi.py

```diff
@@ -1487,7 +1494,7 @@ def replace_special_case(hint: str) -> str:
                     "set_",
                     [
                         "self",
-                        "source: Storage | TypedStorage | UntypedStorage",
+                        "source: Storage | UntypedStorage",
```

Comment:
Removing `TypedStorage` as it is no longer part of public API (to be deprecated)